src/main/webapp/app/overview/course-card.scss

```diff
@@ -1,19 +1,31 @@
 .card {
-    min-height: 320px;
+    height: 283px;
+    min-width: 320px;
+    max-width: 400px;
     transition: transform 0.2s linear;
 
+    @media screen and (max-width: 768px) and (min-width: 478px) {
+        max-width: 550px !important;
+    }
+
+    @media screen and (min-width: 2000px) {
+        max-width: 500px !important;
+    }
+
+    @media screen and (min-width: 2700px) {
+        max-width: 600px !important;
+    }
```

Comment:
Are these values based on trial and error? Do they have a deeper meaning?

Would it make sense to add an explanation here?

Reply:
Not needed anymore, will remove them 👍